project_template/src/index.js

```diff
@@ -4,6 +4,7 @@ import { Provider } from "preact-redux";
 import FastClick from "fastclick";
 import "src/config/sentry";
 import store from "./store";
+import "@babel/polyfill";
```

Comment:
Maybe we could add a comment here pointing to the issue you created? So we instruct users that depending on your use cases you might want to remove this explicit polyfill import.

Reply:
I think it's cleaner to add this here (and rephrase the subtitle a little): https://github.com/kabisa/maji/blob/master/docs/README.md#components-you-might-not-need.

Reply:
Great! 👍